cockatrice/src/chatview/chatview.cpp

```diff
@@ -428,22 +411,22 @@ bool ChatView::isModeratorSendingGlobal(QFlags<ServerInfo_User::UserLevelFlag> u
     QStringList getAttentionList;
     getAttentionList << "/all"; // Send a message to all users
 
-    if (getAttentionList.contains(message) && (userLevel & ServerInfo_User::IsModerator || userLevel & ServerInfo_User::IsAdmin))
-        return true;
+    return (getAttentionList.contains(message)
+            && (userLevel & ServerInfo_User::IsModerator
```

Comment:
We can just return isModerator as isAdmin is a higher level that will return the same thing anyway.

is Admin implies is Moderator